pkg/app/src/components/Account/general-layout.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Container } from '@mui/material'
```

Comment:
please sort dependencies generally from outside to inside, e.g. react at the top, detailed component at the bottom

Reply:
created a PR for this issue #158